docs/source/ingest/destination_connectors/astra.rst

```diff
@@ -0,0 +1,32 @@
+Astra
+===========
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs and embeddings locally on your filesystem and upload those to a Astra DB index.
+
+First you'll need to install the Pinecone dependencies as shown here.
```

Comment:
Pinecone? Missed on copy paste?

Reply:
@ryannikolaidis yep. Rushed out the documentation.